core/source/api/opencmiss/zinc/fieldmatrixoperators.hpp

```diff
@@ -37,14 +37,15 @@ class FieldDeterminant : public Field
 class FieldEigenvalues : public Field
 {
 private:
-	// takes ownership of C handle, responsibility for destroying it
-	explicit FieldEigenvalues(cmzn_field_id field_id) : Field(field_id)
-	{	}
 
 	friend FieldEigenvalues Fieldmodule::createFieldEigenvalues(const Field& sourceField);
```

Comment:
Technically, the friend is not needed any more. However, it's quite nice on the C++ API Doxygen to see the friend to know which factory method to use to create one. Perhaps we need to put a @see Fieldmodule::createFieldTYPE on each class's documentation? (Would require it being transferred from C.)